services/director/src/simcore_service_director/registry_proxy.py

```diff
@@ -410,11 +410,17 @@ async def get_image_details(
     if not labels:
         return image_details
     for key in labels:
-        if not key.startswith("io.simcore."):
+        if not key.startswith("io.simcore.") and not key.startswith(
+            "simcore.service."
+        ):  # Keeping "simcore.service." adds additonally input_paths, output_paths, state_paths
```

Comment:
I would add some tests on these conditions. 
how can we guarantee these heuristics are not discarding other services or future services?
is there any other way e.g. via intergration-version to more reliably identify "legacy dynamic-service"?